lighthouse-core/report/html/renderer/performance-category-renderer.js

```diff
@@ -76,21 +65,17 @@ class PerformanceCategoryRenderer extends CategoryRenderer {
       return element;
     }
 
-    const displayValue = Util.formatDisplayValue(audit.result.displayValue);
+    // Overwrite the displayValue with opportunity's wastedMs
+    const displayEl = this.dom.find('.lh-audit__display-text', element);
     const sparklineWidthPct = `${summaryInfo.wastedMs / scale * 100}%`;
-    const wastedMs = Util.formatSeconds(summaryInfo.wastedMs, 0.01);
-    const auditDescription = this.dom.convertMarkdownLinkSnippets(audit.result.description);
-    this.dom.find('.lh-load-opportunity__sparkline', tmpl).title = displayValue;
-    this.dom.find('.lh-load-opportunity__wasted-stat', tmpl).title = displayValue;
-    this.dom.find('.lh-sparkline__bar', tmpl).style.width = sparklineWidthPct;
-    this.dom.find('.lh-load-opportunity__wasted-stat', tmpl).textContent = wastedMs;
-    this.dom.find('.lh-load-opportunity__description', tmpl).appendChild(auditDescription);
-
-    // If there's no `type`, then we only used details for `summary`
-    if (details.type) {
-      const detailsElem = this.detailsRenderer.render(details);
-      detailsElem.classList.add('lh-details');
-      element.appendChild(detailsElem);
+    this.dom.find('.lh-sparkline__bar', element).style.width = sparklineWidthPct;
+    displayEl.textContent = Util.formatSeconds(summaryInfo.wastedMs, 0.01);
```

Comment:
feels weird to overwrite `Error!` with this in the error case. Will it always have a `summaryInfo` if there was a gatherer or audit error?